runtime/vm/JFRConstantPoolTypes.hpp

```diff
@@ -822,8 +822,11 @@ class VM_JFRConstantPoolTypes {
 	 */
 	static void freeJVMInformationEvent(J9JavaVM *vm)
 	{
-		PORT_ACCESS_FROM_JAVAVM(vm);
-		j9mem_free_memory(getJFRConstantEvents(vm)->JVMInfoEntry.jvmArguments);
+		JFRConstantEvents *jfrConstantEvents = getJFRConstantEvents(vm);
```

Comment:
the NULL check is redundant, `j9mem_free_memory` already does this

Reply:
`jfrConstantEvents` could be `NULL` if `JFR.dump/stop` command are issued w/o `JFR.start` which invokes `initializeJFR()`.
https://github.com/eclipse-openj9/openj9/blob/4b219a2e9748331bf5cc001138c93b9edb7a952d/runtime/vm/jfr.cpp#L587-L588
`j9mem_free_memory` only checks a `NULL` for `jfrConstantEvents->JVMInfoEntry.jvmArguments`.